### PR TITLE
Design

### PR DESCRIPTION
**Pull request by moony-inc**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,12 +2,31 @@
...
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600&display=swap"
```
Comment (vyadl) [inline public/index.html:23]: how much do we use?

File: src/components/App.vue [hunk 2]
```diff
@@ -33,18 +33,17 @@ export default {
...
+      theme: 'theme',
     }),
   },
   created() {
     axiosSettings.initAxios();
     this._setUserFromLocalStorage();
 
     if (this.isLoggedIn) {
-      this._fetchListsForUser();
+      setTimeout(this._fetchListsForUser, 500);
```
Comment (vyadl) [inline src/components/App.vue:46]: oh, we need to test it later (after release) closely

File: src/components/list/ListItem.vue [hunk 3]
```diff
@@ -23,37 +25,35 @@ export default {
...
+          ${Math.floor(Math.random() * (widthWindow - 70 - 5) + 5)}px, 
+          ${Math.floor(Math.random() * heightWindow)}px)`;
 
         styleObj = {
           transform: `${scaleStyle} ${rotateStyle} ${translateStyle}`,
         };
       }
 
-      return this.isCloudModeOn || this.isStarsModeOn ? styleObj : {};
+      return this.mode === 'cloud' || this.mode === 'stars' ? styleObj : {};
```
Comment (vyadl) [inline src/components/list/ListItem.vue:50]: ['cloud', 'stars'].includes(this.mode)

File: src/components/mainPages/Sidebar.vue [hunk 4]
```diff
@@ -152,8 +172,24 @@ export default {
...
+    &.invert-theme {
+      border-left: 1px solid map-get($colors, 'gray-light');
+      background-color: map-get($colors, 'black');
+      color: map-get($colors, 'white');
+
+      &.show {
+        box-shadow: none;
+      }
+
+      .mode-button{
```
Comment (vyadl) [inline src/components/mainPages/Sidebar.vue:188]: space before {

File: src/components/mainPages/Sidebar.vue [hunk 5]
```diff
@@ -152,8 +172,24 @@ export default {
...
+        &::before{
```
Comment (vyadl) [inline src/components/mainPages/Sidebar.vue:189]: and here the same space {

File: src/components/modals/ModalBasic.vue [hunk 6]
```diff
@@ -1,10 +1,13 @@
 <template>
   <modal
     class="modal-basic"
+    :class="{ 'invert-theme': isInvert }"
+    :name="name"
     height="auto"
     :width="width"
-    :name="name"
+    :scrollable="true"
```
Comment (vyadl) [inline src/components/modals/ModalBasic.vue:8]: it can be just scrollable

File: src/components/sidebarContent/ListVisualization.vue [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+      },
+      immediate: true,
+    },
+    theme: {
+      handler: function themeHandler() {
+        this.localTheme = this.theme;
+      },
+      immediate: true,
+    },
+  },
```
Comment (vyadl) [inline src/components/sidebarContent/ListVisualization.vue:99]: let's try to change it to setting ":checked" attr

File: src/components/sidebarContent/TestData.vue [hunk 8]
```diff
@@ -31,32 +34,41 @@
...
+  computed: {
+    ...mapGetters({
+      lists: 'lists',
+    }),
+  },
   created() {
     axios.get('/test_data.json').then(({ data }) => {
       this.testData = data;
     });
+    this.areTestListsShown = Boolean(!this.lists.length);
```
Comment (vyadl) [inline src/components/sidebarContent/TestData.vue:63]: !! instead of Boolean
if you want

Comment (vyadl) [reply src/components/sidebarContent/TestData.vue:63]: hm, no after !this.lists.length it's already Boolean. ! is an operator which return Boolean

File: src/store/actions.js [hunk 9]
```diff
@@ -6,12 +6,9 @@ export default {
 
     if (currentListId) {
       commit('setCurrentListId', currentListId);
-      dispatch('_fetchListById', currentListId);
+      dispatch('_fetchListById', { id: currentListId, token: null });
```
Comment (vyadl) [inline src/store/actions.js:9]: let's rename it to cancelToken - that it will no be cofusing with access token

File: src/store/actions.js [hunk 10]
```diff
@@ -149,29 +148,33 @@ export default {
...
+    if (sorting === 'default') {
+      commit('setMode', 'list');
+    }
   },
-  _switchCloudMode({ commit }) {
-    commit('switchCloudMode');
+  _setMode({ commit }, mode) {
+    commit('setMode', mode);
+
+    if (mode === 'cloud' || mode === 'stars') {
```
Comment (vyadl) [inline src/store/actions.js:170]: use here array and includes

File: src/store/actions.js [hunk 11]
```diff
@@ -149,29 +148,33 @@ export default {
...
+      commit('setSorting', 'shuffle');
+    } else {
+      commit('setSorting', 'default');
```
Comment (vyadl) [inline src/store/actions.js:173]: and it also can be ternary in passing second argument

Comment (moony-inc) [reply src/store/actions.js:173]: No problem, but it's much less obvious

**Pull request merged**